data/Minecraft_Pi_Reborn_Client

```diff
@@ -0,0 +1 @@
+https://jenkins.thebrokenrail.com/job/minecraft-pi-reborn/job/master/lastSuccessfulBuild/artifact/out/minecraft-pi-reborn-client-2.3.2-amd64.AppImage
```

Comment:
This link will break when a new version is released, because it specifically refers to the last successful build.